helm/reana/values.yaml

```diff
@@ -32,6 +32,7 @@ secrets:
   gitlab: {}
   cern:
     sso: {}
+  reana: {}
```

Comment:
Can you add this new config to the [README](https://github.com/reanahub/reana/blob/master/helm/reana/README.md)?